### PR TITLE
Compatible with Hadoop 2.6.0-cdh5.11.0

### PR DESCRIPTION
**Pull request by zuston**

### CODE REVIEW

**Comment by zuston** [comment]
I found the latest TonY code can't be compatible with Hadoop 2.6.0-cdh5.11.0. Fix it @oliverhu

**Comment by oliverhu** [comment]
@zuston is there a use case for this? We can create a release branch for 2.6, but keep the master up with 2.10+

**Comment by zuston** [comment]
We use the Hadoop 2.6.0 with TonY 0.3.20 version, it works well. And it's necessary to keep compatible with Hadoop 2.6.0, since importing some higher level api(like `ResourceUtils`), this leads to incompatibility.
This PR just to fix this. I think no need to create extra release branch.

File: tony-core/src/main/java/com/linkedin/tony/util/Utils.java [hunk 1]
```diff
@@ -211,6 +210,15 @@ public static void setCapabilityGPU(Resource resource, int gpuCount) {
...
   }
 
+  /**
+   * Uses reflection to get memory size.
+   * @param resource  the request container resource
+   * @return the memory size
+   */
+  public static long getMemorySize(Resource resource) {
+    return HadoopCompatibleAdapter.getMemorySize(resource);
+  }
```
Comment (oliverhu) [inline tony-core/src/main/java/com/linkedin/tony/util/Utils.java:220]: remove these functions from utils? just call the compatible adapter from code is good enough.

**Review by oliverhu — APPROVED** [review]

**Pull request merged**